utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -292,18 +372,35 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
 
     int resize_w = width, resize_h = height; // height and width
 
-    RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false);
+    // RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false);
+    RocalImage image0 = input1;
 
     RocalImage image1;
-
+    std::string interpolationTypeName,scaling_node_Name;
+    RocalResizeInterpolationType interpolationType = RocalResizeInterpolationType::ROCAL_LINEAR_INTERPOLATION;
+    RocalResizeScalingMode scale_mode = RocalResizeScalingMode::ROCAL_SCALING_MODE_STRETCH;
```

Comment:
Add a single line after this

Reply:
Do we need interpolation_type and scale_mode here?  move to case 0?